### ISSUE TITLE
Disable Upload/Browse

### ISSUE THREAD

**Issue by brianfreytag** [bahriddin/ckeditor-html5-video#28]
Is there any way to disable the upload and browse functionality on this particular widget individually? I would like my people to be able to upload/browse images, but not upload/browse videos. 

We want to have control over where and when videos are uploaded and then just use this plugin as an easy way for them to paste the URL in to create a clean video tag and configure their autoplay/downloadable/etc.

**Comment by bahriddin** [comment]
If I understood correctly, you just want to use the plugin to manage how to play existing videos (by pasting URL). Is that correct?

**Comment by brianfreytag** [comment]
Correct. That would be the only options. The "Upload" tab and "Browse Server" buttons would not be available to the end users.

To clarify, we would upload our videos to our secure S3 bucket. Then we would use this plugin to paste the URL to it. I have backend code in PHP to replace the links to S3 with their presigned request query parameters for our authenticated users.

We don't want any user to be able to upload videos and other media at their leisure because our user base is known to abuse stuff like that lol. This is for an internal, homespun, ERP system we develop in house. We allow them to upload pictures and we find that there are 9 or 10 duplicates because they upload before their image is finished and whatever. It's a mess lol I really don't want to deal with it with 800MB mp4s
